etc/behat/services.xml

```diff
@@ -30,6 +30,7 @@
         <parameter key="sylius.behat.page.admin.crud.create.class">Sylius\Behat\Page\Admin\Crud\CreatePage</parameter>
         <parameter key="sylius.behat.page.admin.crud.update.class">Sylius\Behat\Page\Admin\Crud\UpdatePage</parameter>
         <parameter key="sylius.behat.service.accessor.notification_accessor.class">Sylius\Behat\Service\Accessor\NotificationAccessor</parameter>
+        <parameter key="sylius.behat.service.notification_validator.class">Sylius\Behat\Service\NotificationValidator</parameter>
```

Comment:
:+1: for services names. Asserter doesn't sound good for me :smile: I would leave Validator :)